_attributes/common-attributes.adoc

```diff
@@ -132,3 +132,5 @@ endif::[]
 :ibmzProductName: IBM Z
 // Red Hat Quay Container Security Operator
 :rhq-cso: Red Hat Quay Container Security Operator
+:sno: single-node Openshift
+:sno-first: Single-node Openshift
```

Comment:
Other common attributes that end with `-first` typically resolve to the full version of a product name, with the same attribute minus `-first` resolving to a shorter version of the product name. In that context, attributes ending with `-first` are typically for first time use in a section. Given this, when I first saw the attribute name `sno-first` in this PR, my assumption was that the attribute was for first time use in a section. However, I now realise that the attribute is intended for use at the start of a sentence.

You could consider using something other than `-first` in the attribute name, to avoid this possible misinterpretation. For example, you could call the attribute something like `sno-capital`.

Reply:
updated.